CMakeLists.txt

```diff
@@ -83,6 +83,14 @@ message("🐍 Found Python ${Python3_VERSION}, ${PYTHON_COMMAND}")
 
 # ======== Python Checks ========
 
+message("⚙️ Run Clang-Formatting for ${PLATFORM}-${TARGET} build target")
+set(PYTHON_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/scripts/clang_format/fix_formatting.py)
+
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -E env PYTHONPATH=${CMAKE_CURRENT_SOURCE_DIR}  ${PYTHON_COMMAND}  ${PYTHON_SCRIPT}
```

Comment:
why not just `${PYTHON_COMMAND} ${PYTHON_SCRIPT}`?

Reply:
Just in case the script relied on anything in the cmake context but i realize we dont rly need all the extra stuff I'll simplify it